distribution/docker/build.gradle

```diff
@@ -266,31 +303,39 @@ void addUnpackDockerContextTask(Architecture architecture, DockerBase base) {
 }
 
 
-private List<String> generateTags(DockerBase base) {
-  String version = VersionProperties.elasticsearch
+private static List<String> generateTags(DockerBase base) {
+  final String version = VersionProperties.elasticsearch
+
+  String image = "elasticsearch${base.suffix}"
+
+  String namespace = 'elasticsearch'
+  if (base == DockerBase.CLOUD || base == DockerBase.CLOUD_ESS) {
```

Comment:
Is it strictly necessary that we do this here? Isn't this done by release manager?

Reply:
As far as I can tell, the release-manager build file just contains what image to pick up, it doesn't do the creation or tagging.